OptimizelySDK/Optimizely.cs

```diff
@@ -308,7 +308,8 @@ public Variation GetVariation(string experimentKey, string userId, UserAttribute
             };
 
             userAttributes = userAttributes ?? new UserAttributes();
-            NotificationCenter.SendNotifications(NotificationCenter.NotificationType.Decision, DecisionInfoTypes.EXPERIMENT, userId,
+            var decisionNotificationType = Config.IsFeatureExperiment(experiment.Id) ? DecisionNotificationTypes.FEATURE_TEST : DecisionNotificationTypes.AB_TEST;
```

Comment:
Can we change it instead of returning `true` / `false`, we should return directly the DecisionNotificationTypes.
Because I assume, no need to create a function to check only Key exist in dictionary or not, which can directly do it here. 
@aliabbasrizvi @mikeng13

Reply:
I think this is fine. No need to expose the map from the project config object if we don't have to. And it's consistent with what we did on the JS SDK.